websites/B/BreathFM/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "846282107462352927",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData;
+	if ("mediaSession" in navigator && navigator.mediaSession.metadata !== null) {
+		presenceData = {
+			largeImageKey: navigator.mediaSession.metadata.artwork[0].src,
+			details: navigator.mediaSession.metadata.title,
+			state: navigator.mediaSession.metadata.artist,
```

Comment:
Also slice the strings for if they are longer than 128 characters (discord's details/state length limits)
```suggestion
			details: navigator.mediaSession.metadata.title.slice(0, 128),
			state: navigator.mediaSession.metadata.artist.slice(0, 128),
```

Reply:
This functionality is native for details and state in the extension/app. There isn't a need to add it here.

Reply:
> This functionality is [native](https://github.com/PreMiD/Extension/blob/5e9894de2c5e82d9425123ab353859158413e65d/src/util/functions/setActivity.ts#L31-L40) for details and state in the extension. There isn't a need to add it here.

Good to know, cuz muliple presences do this.